native-code/android/index.md

```diff
@@ -77,6 +77,33 @@ For instructions on how to build and run, see
 [webrtc/examples/androidapp/README][3].
 
 
+### Using Android Studio
+
+  1. Build the project normally:
+
+     ~~~~~ bash
+     ninja -C out/Debug AppRTCMobile
```

Comment:
Since you write "build the project normally" you assume out/debug builds are the norm, it's not according to https://webrtc.org/native-code/development/. 

@hkjellander WDYT?